lib/modules/datasource/metadata-manual.ts

```diff
@@ -59,6 +81,26 @@ export const manualChangelogUrls: Record<string, Record<string, string>> = {
 // Use this object to define manual source URLs for packages
 // Only necessary if the datasource is unable to locate the source URL itself
 export const manualSourceUrls: Record<string, Record<string, string>> = {
+  maven: {
+    'org.spockframework:spock-core': 'https://github.com/spockframework/spock/',
+    'org.spockframework:spock-bom': 'https://github.com/spockframework/spock/',
+    'org.spockframework:spock-groovy2-compat':
+      'https://github.com/spockframework/spock/',
+    'org.spockframework:spock-guice':
+      'https://github.com/spockframework/spock/',
+    'org.spockframework:spock-junit4':
+      'https://github.com/spockframework/spock/',
+    'org.spockframework:spock-specs':
+      'https://github.com/spockframework/spock/',
+    'org.spockframework:spock-spring':
+      'https://github.com/spockframework/spock/',
+    'org.spockframework:spock-tapestry':
+      'https://github.com/spockframework/spock/',
+    'org.spockframework:spock-testkit':
+      'https://github.com/spockframework/spock/',
+    'org.spockframework:spock-unitils':
+      'https://github.com/spockframework/spock/',
+  },
```

Comment:
Btw, is is possible to override `manualChangelogUrls` and `manualSourceUrls` in `renovate.json`?

Reply:
No